src/form/ConfigurationForm.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Accordion,
+  AccordionSet,
+  Col,
+  ExpandAllButton,
+  Row,
+} from '@folio/stripes/components';
+import stripesFinalForm from '@folio/stripes/final-form';
+import { FormApi } from 'final-form';
+import React, { FormEvent, MutableRefObject, useCallback } from 'react';
+import { FormRenderProps, useField } from 'react-final-form';
+import { FormattedMessage } from 'react-intl';
+import FormValues from '../types/FormValues';
+import AggregateMenu from './sections/AggregateMenu';
+import CriteriaMenu from './sections/CriteriaMenu';
+import DataTokenMenu from './sections/DataTokenMenu';
+import ExportPreview from './sections/ExportPreview';
+import HeaderFooterMenu from './sections/HeaderFooterMenu';
+import SchedulingMenu from './sections/SchedulingMenu';
+import TransferInfoMenu from './sections/TransferInfoMenu';
+
+export const FORM_ID = 'ui-plugin-bursar-export-form';
+
+interface ConfigurationFormProps {
+  formApiRef: MutableRefObject<FormApi<FormValues> | null>;
+}
+
+function ConfigurationForm({
```

Comment:
why do you separate `form` as a separate layer in project structure? what is the difference between form and components in your case? it seems that your components are not really reusable and connected to specific sections (I can be wrong now as I haven't spent enough time on it)

I'm just trying to understand your approach,

Reply:
This module won't be reused, it's only for this module.

Reply:
it doesn't matter, your components are used almost once, but they live in components